### DISCUSSION TITLE
Add epoch or precision

### DISCUSSION THREAD

**Discussion by jramram** [offa/influxdb-cxx#230] [Q&A] [answered]
Hi, 
I want to add in some queries the parameter epoch, but I cannot find how I can make it (using the last version). 
I was looking at the code, but I think it is not possible. Is it?

According to documentation

https://docs.influxdata.com/influxdb/v2/reference/api/influxdb-1x/query/#epoch

```
curl --get http://localhost:8086/query \
  --header "Authorization: Token INFLUX_API_TOKEN" \
  --data-urlencode "db=mydb" \
  --data-urlencode "rp=myrp" \
  --data-urlencode "q=SELECT used_percent FROM mem WHERE host=host1" \
  --data-urlencode "epoch=ms"
```

It should be added as a parameter, but in the code 
[http.cxx
](https://github.com/offa/influxdb-cxx/blob/4b91200a5d0631eeb11585dba7a7d1c0a1a490e2/src/HTTP.cxx#L85)

```
session.SetUrl(cpr::Url{endpointUrl + "/query"});
session.SetParameters(cpr::Parameters{{"db", databaseName}, {"q", query}});
```
Only db and query parameters are available

Thanks in advance

**Comment by offa** [comment, marked as answer]
Epoch queries aren't supported yet. I have created an issue for a new feature #231.

**Reply by jramram** [reply]
Thanks !
